test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Agent/AgentNotebookTests.cs

```diff
@@ -0,0 +1,128 @@
+using System;
```

Comment:
using statements should be sorted.  The convention in this code base is (1) System imports first alphabetically then (2) everything else alphabetically (3) the using static can be last.